tools/README.txt

```diff
@@ -1137,3 +1137,22 @@ zipme.sh
         Same as above but use the key-id XXXXXX to sign the tarballs
       ./tools/zipme.sh -e "*.swp tmp" 9.0.0
         Create the tarballs but exclude any .swp file and the "tmp" directory.
+
+gitver.sh
```

Comment:
It seem that version.sh can generate the similar version info like gitver.sh, can we reuse version.sh?

Reply:
This computes the version in an importantly different way since it take a minimum version and uses tag information to find a suitable version string.

Reply:
version.sh has the similar logic to generate version from tag, if the current method isn't good, it's better to replace with your new appraoch there.